src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -2,9 +2,9 @@
 
 import com.wolox.training.models.Book;
 import java.util.Optional;
-import org.springframework.data.repository.Repository;
+import org.springframework.data.repository.CrudRepository;
 
-public interface BookRepository extends Repository<Book, Long> {
+public interface BookRepository extends CrudRepository<Book, Long> {
 
     Optional<Book> findByAuthor(String author);
```

Comment:
This is from a previous PR, but when using `findByAuthor(String author)` a collection is returned. So what happens if you have more than one book with the same author? (It's not a rhetorical question, I want you to try it 😋)

Reply:
```"query did not return a unique result: 2; nested exception is javax.persistence.NonUniqueResultException: query did not return a unique result: 2",
    "trace": "org.springframework.dao.IncorrectResultSizeDataAccessException: query did not return a unique result: 2; nested exception is javax.persistence.NonUni```
![image](https://user-images.githubusercontent.com/17320168/79802818-b5cbfb80-832e-11ea-808a-064f0c657697.png)

Reply:
![image](https://user-images.githubusercontent.com/17320168/79802899-e318a980-832e-11ea-9e2d-68218809afda.png)

Reply:
think I manage to fix it. Take a look